server/src/main/java/org/tctalent/server/logging/LogField.java

```diff
@@ -51,7 +51,7 @@
 public enum LogField {
 
   USER_ID("uid", 100),
-  CANDIDATE_NUMBER("cn", 101),
+  CANDIDATE_ID("cid", 101),
```

Comment:
I note that the JavaDoc is not updated.

However, the long term intention is to eventually convert candidate numbers to numbers and make them the primary key - dispensing with candidate ids altogether. Candidate numbers are all users ever see - candidate ids are for internal use only at the moment. It is often convenient to see candidate numbers in logs - given that is what typically displays.

Reply:
Discussed in team meeting. Agreed to use candidate ID for logging in the service code but can easily extend to support candidate number as an additional log field if/when needed. Currently, candidate number is not logged anywhere by the service code.

Separately, a ticket has been opened to follow up on the future intention for using candidate number, which is currently a String.

Lastly, the JavaDoc is tidied up in my next PR connected to issue 1205.